DSharpPlus/Enums/Permission.cs

```diff
@@ -68,7 +68,7 @@ public enum Permissions : long
         /// Indicates all permissions are granted
         /// </summary>
         [PermissionString("All permissions")]
-        All = 2147483647,
+        All = 4294967295,
```

Comment:
```suggestion
        All = 6441926655,
```

Thing here is, i tried to calculate the perms together.
the old perms, the new perms and the one i add are all different from yours. could it be that i msised something?
please correct if wrong.